packages/x-date-pickers-pro/src/MobileDateRangePicker/MobileDateRangePicker.test.tsx

```diff
@@ -57,6 +57,7 @@ describe('<MobileDateRangePicker />', () => {
     clock,
     componentFamily: 'legacy-picker',
     views: ['day'],
+    mode: 'mobile',
```

Comment:
```suggestion
    variant: 'mobile',
```

To match the internal nomenclature